test/OrchardCore.Tests/Apis/Context/Extensions/HttpRequestExtensions.cs

```diff
@@ -102,7 +102,7 @@ internal static class HttpRequestExtensions
             JsonSerializerSettings settings = null)
         {
             var content = new StringContent(
-                JsonConvert.SerializeObject(value, settings ?? JsonSettings),
+                JsonConvert.SerializeObject(value, settings ?? _jsonSettings),
                 Encoding.UTF8,
                 "application/json");
```

Comment:
Should we store this string as a constant since it's reused? I understand if you want that is a separate PR

Reply:
Yes good idea, the same string is used in 22 other places. There are rules that e.g. suggest a const when a given string appears more than 3 times in a given file, but not part of our current rules.

Note: These values can already be retrieved from the `FileExtensionContentTypeProvider` aspnetcore class, but that need to be injected through `IContentTypeProvider`.